src/lib/controller.ts

```diff
@@ -121,14 +127,26 @@ export class Controller {
     }
   };
 
+  private metrics = async (req: express.Request, res: express.Response) => {
+    try {
+      res.send(await this.metricsCollector.getMetrics());
+    } catch (err) {
+      console.error(err);
+      res.status(500).send("Internal Server Error");
+    }
+  };
+
   private mutate = async (req: express.Request, res: express.Response) => {
+    const startTime = performance.now();
```

Comment:
would probably be better to just use the request time tracking in the `logger` function